views/js/runner/plugins/tools/textToSpeech/textToSpeech.js

```diff
@@ -349,7 +365,7 @@ define([
                     max: 100
                 },
                 start: speed,
-                step: 10
+                step: 15
```

Comment:
What does this value mean? Could you explain it with a comment?

Reply:
Yep, this is the part of the changes requested by textHelp guys. Actually, this value is the step for speed slider. Now the step will be 15. So, values that can be chosen with help of slider are: 0, 15, 30, etc., etc., 100. Some time later, also "border values" of the slider will be fixed, as textHelp requested from us that only some speeds are available to choose to end-user - it's for caching purposes.

Reply:
I meant: please put a comment in the code to explain the meaning of the value, for other developers this will be helpful :)

Reply:
Awwww, I understand, will comment it )